### PR TITLE
Added high and low limits for discharge pressures and temperatures

### PR DESCRIPTION
**Pull request by wroantree-willow**

### CODE REVIEW

File: Ontology/Willow/Capability/Parameter/Limit/Quantity/Pressure/DischargeAirStaticPressureSetpointHighLimit.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "@id": "dtmi:com:willowinc:DischargeAirStaticPressureSetpointHighLimit;1",
+  "@type": "Interface",
+  "displayName": {
+    "en": "Discharge Air Static Pressure Setpoint High Limit"
+  },
+  "description": {"en": "The high limit associated with discharge (supply) air static pressure"},
+  "extends" : [
+    "dtmi:com:willowinc:DischargeAirStaticPressureSetpoint;1"
```
Comment (rszcodronski) [inline Ontology/Willow/Capability/Parameter/Limit/Quantity/Pressure/DischargeAirStaticPressureSetpointHighLimit.json:9]: Should this extend a Setpoint or a Limit? If limits are defining the acceptable range for a setpoint, maybe they should inherit from Limit rather thatn Setpoint.

File: Ontology/Willow/Capability/Parameter/Limit/Quantity/Pressure/DischargeAirStaticPressureSetpointHighLimit.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "@id": "dtmi:com:willowinc:DischargeAirStaticPressureSetpointHighLimit;1",
+  "@type": "Interface",
+  "displayName": {
+    "en": "Discharge Air Static Pressure Setpoint High Limit"
+  },
+  "description": {"en": "The high limit associated with discharge (supply) air static pressure"},
```
Comment (rszcodronski) [inline Ontology/Willow/Capability/Parameter/Limit/Quantity/Pressure/DischargeAirStaticPressureSetpointHighLimit.json:7]: Can we be a bit more concise in the description? In reading this, I'm not sure if it is defining a limit on the discharge air static pressure sensor or setpoint.

Comment (wroantree-willow) [reply Ontology/Willow/Capability/Parameter/Limit/Quantity/Pressure/DischargeAirStaticPressureSetpointHighLimit.json:7]: Agreed, I'll make an update

File: Ontology/Willow/Capability/Parameter/Limit/Quantity/Pressure/DischargeAirStaticPressureSetpointHighLimit.json (hunk 1 above)
Comment (wroantree-willow) [reply Ontology/Willow/Capability/Parameter/Limit/Quantity/Pressure/DischargeAirStaticPressureSetpointHighLimit.json]: Agreed, for ease of use with analytics, I've removed the setpoint extensions

**Review by rszcodronski — APPROVED** [review]

**Pull request merged**